android/src/fdroid/play/release-notes/en-US/default.txt

```diff
@@ -1,14 +1,10 @@
-• New OSM data as of November 19
-• Correct bicycle's speed for routing
-• Add Mastodon in "?"
-• Fix some crashes
-• Fix navigation menu under navbar
-• Fix distance not shown on search results
-• Fix some overlapping icons
-• Add highway=busway and physiotherapist
-• Islands and archipelagos on World Map
-• Fix water/peer and waterway=dam drawing priority
-• Fix leisure areas
-• Add Estonian, update cs, es, nb, tr translation
-• Fix RV and Catalan categories
-…more: omaps.app/news
+• New OpenStreetMap data as of December 14, 2022
+• Fixed crash in screen split mode & on screen rotation
+• Supported KML 2.3 with multiple tracks from OpenTracks
+• Fine-tuned routing weights for all types of roads
+• Eliminated annoying dialogs when location is disabled
```

Comment:
Fixed annoying "Enable Location" dialogs

Reply:
The problem existed only when location was off. This release affects no other cases.

Reply:
No. The problem existed when the location was ON, but with either GPS-only or wifi/cellular-only location providers enabled.